frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -331,6 +342,7 @@ local option_titles = {
     chapter_time_to_read = _("Chapter time to read"),
     frontlight = _("Frontlight level"),
     mem_usage = _("KOReader memory usage"),
+    wifi_status = _("WiFi status"),
```

Comment:
Note that while I'm a proponent of wifi as a regular word (without any weird brand name stylization), the spelling we use is `Wi-Fi`.